src/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs

```diff
@@ -15,6 +15,8 @@ public static class DotnetFiles
         /// </summary>
         public static string VersionFile => Path.GetFullPath(Path.Combine(typeof(DotnetFiles).GetTypeInfo().Assembly.Location, "..", ".version"));
 
+        public static string NuGetPackagesArchive => Path.GetFullPath(Path.Combine(typeof(DotnetFiles).GetTypeInfo().Assembly.Location, "..", "nuGetPackagesArchive.lzma"));
```

Comment:
It would be nice to encapsulate the path into its own private variable (or static method to return it), that way it isn't duplicated here and they will never get out of sync.